pkg/jmxfetch/jmxfetch.go

```diff
@@ -186,7 +186,9 @@ func (j *JMXFetch) Start(manage bool) error {
 	subprocessArgs = append(subprocessArgs, j.Command)
 
 	if j.JmxExitFile != "" {
-		j.exitFilePath = filepath.Join(here, "dist", "jmx", j.JmxExitFile) // FIXME : At some point we should have a `run` folder
+		exitFileDir := filepath.Join(here, "agent", "dist", "jmx", "run")
```

Comment:
I don't think we were referring to a `run` dir under `dist` though...

Reply:
When replicating posix-like directory structures in Windows, it's customary to create them relative to the install path of your app. In this case, I thought that dist/jmx/ is the "install path" of jmxfetch. I'm happy to change it if you can think of an alternative.

What about simply using the agent directory (`here`)? In that case I would still remove the comment saying to use a "run" dir.

Reply:
just to give some context: the `run` directory of the Agent generally refers to the `run/` directory that's created at the root of the Agent installation directory and to which the Agent user should be guaranteed to have write access. Typically that's where the agent writes its PID file on Linux, or other files written at runtime.

On Windows, the Agent also seems to have a `run/` directory under `<program_data>\Datadog\`. @derekwbrown would that be a good place to write that file at runtime? Or is there a "better" directory on Windows where the Agent's user is guaranteed to have write access, and to which it should write run-time files instead?